ietf-optical-impairment-topology.tree

```diff
@@ -99,7 +94,7 @@ module: ietf-optical-impairment-topology
     |        |              +--ro out-of-band-osnr?
     |        |              |       snr
     |        |              +--ro tx-polarization-power-difference?
-    |        |              |       power-in-db
+    |        |              |       power-in-dbm
```

Comment:
if its a power difference it is in db

Reply:
We have removed the power-in-db data type from l0-types: maybe better to defined it as a decimal-2-digits-or-null type with units "dB"

I assume the values can be positive or negative so we do not need to define any range

Reply:
yes decimal-2-digits-or-null type with units "dB" works fine